bigbluebutton-html5/imports/ui/components/presentation/styles.scss

```diff
@@ -19,31 +19,32 @@
 }
 
 .presentationPaper {
-  position: absolute;
-  display: block;
   width: 100%;
-  height: 100%;
+  display: flex;
```

Comment:
A lot of the styles in this file were changed recently, since we switched from pure CSS way of scaling the presentation. Now it's a JavaScript-controlled wrapper and then CSS scaling inside.
Tested the latest - everything is working perfectly in Safari for me (Safari 11).
But maybe I don't know about all the cases that you were trying to fix here.

Reply:
I've removed those changes in favor of the new styles for the presentations, I also tested in safari, and it seems to work :)